### PR TITLE
Added a default template entry grade setting

### PR DESCRIPTION
**Pull request by mjvkeulen**
Added a template setting which can be used to speed up the grading process.

image

image

image

- Fixes Z#1469

### CODE REVIEW

File: src/vue/src/components/template/TemplateEditSettings.vue [hunk 1]
```diff
@@ -21,6 +21,26 @@
...
+
+        <hr/>
+
+        <b-form-input
+            id="default-grade-input"
+            v-model="template.default_grade"
+            type="number"
+            class="theme-input float-right mb-3 ml-3"
+            size="2"
+            autofocus
```
Comment (Larspolo) [inline src/vue/src/components/template/TemplateEditSettings.vue:33]: I would not make this auto focused. If anything, make the title auto focused (or maaybe the first unfilled spot) This looks especially weird when creating a new template

**Review by Larspolo — APPROVED** [review]
Mostly would like to see the autofocus fixed, rest is working nice, and also looks good!

File: src/django/VLE/migrations/0080_default_template_grade.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    dependencies = [
+        ('VLE', '0079_removed_blank_and_null_from_template_chain'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='templatechain',
+            name='default_grade',
+            field=models.FloatField(blank=True, null=True),
```
[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/VLE/migrations/0080_default_template_grade.py:16]: This only applies to char/text fields, where null can be represented by None or ''

[the remark this answers is not in the document]
Comment (Larspolo) [reply src/django/VLE/migrations/0080_default_template_grade.py:16]: Then what does `''` do for float fields?

[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/VLE/migrations/0080_default_template_grade.py:16]: What do you mean? `''` is an invalid data type for a float field?

[the remark this answers is not in the document]
Comment (Larspolo) [reply src/django/VLE/migrations/0080_default_template_grade.py:16]: What does blank do for a float field? I thought `blank == ''`

[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/VLE/migrations/0080_default_template_grade.py:16]: blank is Django's term for allowing a value that represents null. 

So null=True allows null values in the DB, but without blank=True will yield validation errors on data clean, e.g. serialize.is_valid when passed None values.

**Review by engelhamer — CHANGES_REQUESTED** [review]
Looks good, minor tweak needed.

**Review by engelhamer — APPROVED** [review]

**Pull request merged**